flake.nix

```diff
@@ -0,0 +1,17 @@
+{
+  description = "Kubernetes resource builder using nix";
+
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+  inputs.nixpkgs.url = "github:NixOS/nixpkgs";
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let pkgs = nixpkgs.legacyPackages.${system}; in
+      rec {
+        packages.kubenix = pkgs.callPackage ./default.nix {
```

Comment:
maybe better to use `lib` instead of `packages`?

Reply:
`packages` is the one of the [flake special outputs](https://github.com/NixOS/nix/blob/master/src/nix/flake-check.md#evaluation-checks), so this is required for `nix run ...` to work.

Reply:
ok